backend/tests/restrictionTests.test.js

```diff
@@ -0,0 +1,80 @@
+//import axios from 'axios';
+var axios = require('axios')
+
+jest.mock('axios');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
+
+test('Testing add restriction', () => {
+	const req = {'userID': 1, 'dietaryRestrictions': ['apple', 'orange']};
+	const res = {'result': 'Successfully added restriction'};
+
+    axios.put = jest.fn().mockResolvedValue(res);
+    return axios.put(UserStoreURL + '/addRestrictions', {
+        method: 'PUT',
+        headers: {
+            'Content-Type': 'application/json'
```

Comment:
Codacy has a fix for the issue: [Replace `····'Content-Type':·'application/json'` with `"Content-Type":·"application/json",`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
        "Content-Type": "application/json",
```